ibis/backends/flink/registry.py

```diff
@@ -292,10 +296,35 @@ def _array_index(translator: ExprTranslator, op: ops.arrays.ArrayIndex):
     return f"{table_column_translated} [ {index_translated} + 1 ]"
 
 
-def _array_length(translator: ExprTranslator, op: ops.arrays.ArrayLength) -> str:
+def _array_length(translator: ExprTranslator, op: ops.ArrayLength) -> str:
     return f"CARDINALITY({translator.translate(op.arg)})"
 
 
+def _array_position(translator: ExprTranslator, op: ops.ArrayPosition) -> str:
+    arg = translator.translate(op.arg)
+    other = translator.translate(op.other)
+    return f"ARRAY_POSITION({arg}, {other}) - 1"
+
+
+def _array_slice(translator: ExprTranslator, op: ops.ArraySlice) -> str:
+    array = translator.translate(op.arg)
+    start = op.start.value
+    # Note (mehmet): The offsets are 1-based for ARRAY_SLICE.
```

Comment:
```suggestion
    # The offsets are 1-based for ARRAY_SLICE.
```
Nit: Not sure you need to tag yourself in a regular comment

Reply:
Thanks, removed my name.